src/Button.css

```diff
@@ -0,0 +1,10 @@
+.Button {
+  width: 25%;
+  text-align: center;
+  border-left: 1px solid black;
```

Comment:
This looks a little bit weird on the UI, you might as well just leave the default border style, or modify all the borders for a more consistent look, but just modifying the left border looks strange.

Also, since you are using the same color here as the `color` property, you could use [`currentColor`](https://www.w3.org/TR/2018/REC-css-color-3-20180619/#currentcolor)